frontend/src/components/PrescreeningForm.tsx

```diff
@@ -0,0 +1,227 @@
+    } else if (
+      question1Value === "op11" &&
+      question2Value === "op21" &&
+      question3Value === "op31"
+    ) {
+      finalPath = pathFour;
+    } else {
+      finalPath = pathNQ;
+    }
+
+    // setFinalPath(finalPath);
+
+    navigate(finalPath);
+  };
+
+  return (
+    <Page>
+      <h1 className={styles.title}>Prescreening Questions</h1>
+      <p className={styles.headingtwo}>
+        Please answer the following questions about your experience as an Interior Designer. Based
+        on your responses, we will automatically direct you into the appropriate pathway. View more
+        information about the pathways here.
+      </p>
+
+      <form onSubmit={handleFormSubmit} id="form1">
+        {/* Question 1 */}
+        <div>
+          <p className={styles.question}>
+            1. Please select the appropriate amount of experience you currently have.
+          </p>
+
+          <br></br>
+
+          <input
+            type="radio"
+            id="op11"
+            name="question1"
+            checked={question1Value === "op11"}
+            onChange={() => {
+              setQuestion1Value("op11");
+            }}
+          />
+
+          <label htmlFor="op11" className={styles.label}>
+            <p>
+              5+ years of Diversified Design Experience <strong> and/or </strong> 40+ Core Units
+            </p>
+          </label>
+
+          <br></br>
+
+          <input
+            type="radio"
+            id="op12"
+            name="question1"
+            checked={question1Value === "op12"}
+            onChange={() => {
+              setQuestion1Value("op12");
+            }}
+          />
+
+          <label htmlFor="op12" className={styles.label}>
+            <p>
+              I do not have 5 years of Diversified Design Experience <strong> and/or </strong> 40+
+              Core Units but am in the process of completing <strong> one </strong> of these two and
+              will do <strong> within a year </strong> from today.
+            </p>
+          </label>
+
+          <br></br>
+
+          <input
+            type="radio"
+            id="op13"
+            name="question1"
+            checked={question1Value === "op13"}
+            onChange={() => {
+              setQuestion1Value("op13");
+            }}
+          />
+
+          <label htmlFor="op13" className={styles.label}>
+            <p>
+              I do not have 5 years of Diversified Design Experience or have 40+ Core Units and
+              <strong> will not complete </strong> at least one of these two requirements{" "}
+              <strong> within a year </strong> from today.
+            </p>
+          </label>
+        </div>
+
+        {/* Question 2 */}
+        <div>
+          <p className={styles.question}>
+            2. If you have passed any of the following qualifying National Interior Design Exams,
+            please indicate so
+            <ul className={styles.ulprescreening}>
```

Comment:
`<ul>` cannot be inside `<p>` . Close the `<p>` before the ul and then apply the appropriate stylings to the ul element. You can wrap the `<p>` and `<ul>` in a `<div>` and apply `className={styles.question}` to the div instead of `<p>`

Reply:
![image](https://github.com/TritonSE/CCIDC-Application-Overhaul/assets/59126064/c6b6e0f0-5ff5-4b87-bb37-39acb7928af8)
The `<ul>` doesn't seem indented enough

Reply:
The marker seems a bit big you can use the following method to change the size of the marker

```
.ulprescreening > li::marker {
  font-size: 10px;  //Size might not be correct
}
```

what this does is it selects all of the markers of `<li>` that are childrens of `.ulprescreening` and changes its size.